himan-scripts/CB-TCU-cloud.lua

```diff
@@ -10,25 +10,33 @@ local MU = level(HPLevelType.kMaximumThetaE,0)
 local HL = level(HPLevelType.kHeightLayer,500,0)
 local HG = level(HPLevelType.kHeight,0)
 
```

Comment:
"local" could be used for these variables

Reply:
What does "local" actually do?

Reply:
Well it declares the variable as a local variable instead of global, as you may have guessed :D 

Basically it is the way lua recommends to be done: always use "local" unless you have a reason not to. It also has some performance implications as fetching a variable from global scope is more expensive. I have not benchmarked this though and I guess it's rather small, unless running on a very tight loop.